addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -237,6 +265,11 @@ function factory(dependencies) {
         hasComposerFocus: attr({
             related: 'composer.hasFocus',
         }),
+        /**
+         * Determines whether this thread view should show its member list.
+         * Only makes sense if this thread is a channel.
+         */
+        hasMemberList: attr({}),
```

Comment:
Don't think you need empty `{}`

Reply:
I think I need to add a default there actually, I'll add true for now.